### DISCUSSION TITLE
Using Goreleaser to publish a Github Action to the Marketplace

### DISCUSSION THREAD

**Discussion by rocktavious** [goreleaser/goreleaser#3692] [Q&A] [answered]
Hello,

I absolutely love goreleaser so this may be me trying to fit a square peg in a round hole but I have to believe its possible and there is a solution for this.

Background:
I've setup goreleaser on a github action repository (one that we publish to the Github Actions Marketplace) - https://github.com/OpsLevel/report-deploy-github-action

It works up to the point of actually releasing it to the market place - it seems there is a special section added to your "Cut a new Release" page when you have an action.yml file - see here
![Screen Shot 2023-01-10 at 7 37 42 PM](https://user-images.githubusercontent.com/735015/211697886-b9e75ee8-b37c-4b56-9959-1ad9598a82d4.png)

It seems though when you use goreleaser to create the github release it doesn't submit this extra information and your Github Actions marketplace page is not update - the above release was `v0.4.0` but see the following image of our marketplace page its still `v0.3.0`
![Screen Shot 2023-01-10 at 7 39 42 PM](https://user-images.githubusercontent.com/735015/211698119-a49f3bf0-5946-4652-ad2e-0f3bddffb4b3.png)

So first i'm wondering - is it even possible to automate this (even outside of goreleaser - i've not been successful in finding information about this)

Secondly - would the goreleaser project consider adding this as a feature?

If there is another way of doing this while still maintaining all the niceties of having a single go releaser based release pipeline across all our opensource tools i'm open to alternatives but using goreleaser for all our github release needs has been a huge boon for easily setting up a new release process on a new repo.  So thats why i'd like to keep using it here even for a github actions repo.

**Comment by caarlos0** [comment, marked as answer]
sorry, that is out of scope of goreleaser.
